app/assets/javascripts/float-kit/addon/components/d-float-body.gjs

```diff
@@ -0,0 +1,84 @@
+import Component from "@glimmer/component";
+import FloatKitApplyFloatingUi from "float-kit/modifiers/apply-floating-ui";
+import FloatKitCloseOnEscape from "float-kit/modifiers/close-on-escape";
+import FloatKitCloseOnClickOutside from "float-kit/modifiers/close-on-click-outside";
+import { modifier } from "ember-modifier";
+import { getScrollParent } from "float-kit/lib/get-scroll-parent";
+import concatClass from "discourse/helpers/concat-class";
+import { htmlSafe } from "@ember/template";
+import { concat } from "@ember/helper";
+import TrapTab from "discourse/modifiers/trap-tab";
+import DFloatPortal from "float-kit/components/d-float-portal";
+
+export default class DFloatBody extends Component {
+  <template>
+    {{! template-lint-disable modifier-name-case }}
+    <DFloatPortal
+      @inline={{@inline}}
+      @portalOutletElement={{@portalOutletElement}}
+    >
+      <div
+        class={{concatClass
+          @mainClass
+          (if this.options.animated "-animated")
+          (if @instance.expanded "-expanded")
+        }}
+        data-identifier={{this.options.identifier}}
+        aria-labelledby={{@instance.id}}
+        aria-expanded={{if @instance.expanded "true" "false"}}
+        role={{@role}}
+        {{(if @trapTab (modifier TrapTab))}}
+        {{FloatKitApplyFloatingUi this.trigger this.options}}
```

Comment:
nitpick, but I wonder if we could move this up a line? It makes it look like it's a part of the above if condition (I know the above is inline if)